src/AspNet.Security.OAuth.Twitch/TwitchAuthenticationHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://Github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Twitch {
+
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Twitch after a successful authentication process.
+    /// </summary>
+    public static class TwitchAuthenticationHelper {
+
+        /// <summary>
+        /// Gets the identifier corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIdentifier([NotNull] JObject user) => user.Value<string>("_id");
+
+        /// <summary>
+        /// Gets the login corresponding to the authenticated user.
+        /// </summary>
+        public static string GetLogin([NotNull] JObject user) => user.Value<string>("name");
```

Comment:
`GetLogin` uses "name" like `GetName`. Is it really useful to have the same claim twice or is it a bad copy/paste?